modules/shipit_scriptworker/manifests/settings.pp

```diff
@@ -55,32 +59,56 @@
             },
         },
       },
+      'tb-dev'  => {
+        worker_group             => 'shipit-v1',
+        worker_type              => 'tb-shipit-dev',
+        verbose_logging          => true,
+        taskcluster_client_id    => 'project/comm/thunderbird/releng/scriptworker/shipit/dev',
+        taskcluster_access_token => secret('comm_thunderbird_shipit_scriptworker_taskcluster_access_token_dev'),
+        taskcluster_scope_prefix => 'project:comm:thunderbird:releng:ship-it:',
+
+        sign_chain_of_trust      => false,
+        verify_chain_of_trust    => true,
+        verify_cot_signature     => false,
+        cot_product              => 'thunderbird',
+
+        ship_it_instances        => {
+            "project:comm:thunderbird:releng:ship-it:staging" => $ship_it_stage_instance_config,
+        },
+      },
+      'tb-prod'  => {
+        worker_group             => 'shipit-v1',
+        worker_type              => 'tb-shipit-prod',
+        verbose_logging          => true,
```

Comment:
I think this is unused since in modules/shipit_scriptworker/manifests/init.pp you're using the one from line 111 - https://github.com/mozilla/build-puppet/pull/49/files#diff-16fc35f7191a6528caf61089d8af4f3eR111